utils/build/virtual_machine/provisions/installer-auto-inject/auto-inject_installer_script.yml

```diff
@@ -7,6 +7,14 @@
       local_path: utils/build/virtual_machine/provisions/auto-inject/docker/docker_config.yaml
 
   remote-command: |
+
+    #Env variables set on the scenario definition. Write to file and load  
+    SCENARIO_AGENT_ENV="${DD_AGENT_ENV:-''}"
+    echo "${SCENARIO_AGENT_ENV}" > scenario_agent.env
+    echo "AGENT VARIABLES CONFIGURED FROM THE SCENARIO:"
+    cat scenario_agent.env
+    export $(cat scenario_agent.env | xargs) 
```

Comment:
Isn't it easier to use something like `eval`?

Reply:
You are probably right. At this point I was stuck for several hours.